src/main/resources/standardTestCases/multiline_command_line.test

```diff
@@ -0,0 +1,13 @@
+name: multiline_command_line
+testFormat: workflowsuccess
+ignore: true # Fails on JES, Succeeds locally. WTF?
```

Comment:
Thus the ignore currently until it is explored more thoroughly. At the moment in Cromwell this is only being tested against the local backend.